pkg/distro/rhel/images.go

```diff
@@ -487,17 +488,28 @@ func EdgeInstallerImage(workload workload.Workload,
 
 	img.Platform = t.platform
 	img.ExtraBasePackages = packageSets[InstallerPkgsKey]
-	img.Users = users.UsersFromBP(customizations.GetUsers())
-	img.Groups = users.GroupsFromBP(customizations.GetGroups())
+	img.Kickstart = &kickstart.Options{
```

Comment:
(idle wondering) This makes me wonder if we should have a "kickstart.New(customizations *blueprint.Customizations)" ? But maybe they are all too diffrent.

Reply:
I think that might become conditioned on the image type.  It would help clarify the behaviour.

Actually, thinking about it now I realise this would be a good way to centralise all the interactions and consider why they differ if they do.  I wouldn't mind blocking on this.